src/components/index.js

```diff
@@ -1,18 +1,18 @@
-// export { default as Account } from "./Account";
-export { default as Address } from "./Address";
-export { default as AddressInput } from "./AddressInput";
-export { default as Balance } from "./Balance";
-export { default as Blockie } from "./Blockie";
-export { default as BytesStringInput } from "./BytesStringInput";
-export { default as Contract } from "./Contract";
-export { default as EtherInput } from "./EtherInput";
-export { default as Faucet } from "./Faucet";
-export { default as GasGauge } from "./GasGauge";
-export { default as Header } from "./Header";
-export { default as Provider } from "./Provider";
-export { default as Ramp } from "./Ramp";
-export { default as Swap } from "./Swap";
-export { default as ThemeSwitch } from "./ThemeSwitch/ThemeSwitch";
-export { default as Timeline } from "./Timeline";
-export { default as TokenBalance } from "./TokenBalance";
-export { default as Wallet } from "./Wallet";
+// // export { default as Account } from "./Account";
+// export { default as Address } from "./Address";
+// export { default as AddressInput } from "./AddressInput";
+// export { default as Balance } from "./Balance";
+// export { default as Blockie } from "./Blockie";
+// export { default as BytesStringInput } from "./BytesStringInput";
+// export { default as Contract } from "./Contract";
+// export { default as EtherInput } from "./EtherInput";
+// export { default as Faucet } from "./Faucet";
+// export { default as GasGauge } from "./GasGauge";
+// export { default as Header } from "./Header";
+// export { default as Provider } from "./Provider";
+// export { default as Ramp } from "./Ramp";
+// export { default as Swap } from "./Swap";
+// export { default as ThemeSwitch } from "./ThemeSwitch/ThemeSwitch";
+// export { default as Timeline } from "./Timeline";
+// export { default as TokenBalance } from "./TokenBalance";
+// export { default as Wallet } from "./Wallet";
```

Comment:
What if we just deleted these?

Reply:
i think we can. I kept them as a reference jic but I can make a more targeted cleanup card and handle the majority of deletes from that so we dont get too crossed up between issues